Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8238,10 +8251,23 @@ def get_whois(domain, normalized=None):
     return parse_raw_whois(raw_data, normalized=normalized, never_query_handles=False,
                            handle_server=server_list[-1])
 
+
 # Drops the mic disable-secrets-detection-end
 
+def get_domain_from_query(query):
```

Comment:
why not use libraries that to this for you?

Reply:
I've checked 2 libraries (one of them was `tldextract`), they weren't as accurate as expected so I did it manually.